banzai_nres/settings.py

```diff
@@ -0,0 +1,34 @@
+from banzai_nres.images import NRESImage
+from banzai import settings
+from banzai.context import TelescopeCriterion
+import operator
+from banzai import bias, trim, dark, gain, bpm, qc
+import banzai_nres.bias as nres_bias
+import banzai_nres.dark as nres_dark
+from banzai_nres.flats import FlatStacker
+
+NRES_CRITERIA = [TelescopeCriterion('camera_type', operator.contains, 'NRES')]
+
+settings.ORDERED_STAGES = [bpm.BPMUpdater,
+                           qc.HeaderSanity,
+                           qc.ThousandsTest,
+                           qc.SaturationTest,
+                           bias.OverscanSubtractor,
+                           gain.GainNormalizer,
+                           trim.Trimmer,
+                           bias.BiasSubtractor,
+                           dark.DarkSubtractor]
+
+BIAS_IMAGE_TYPES = ['BIAS']
+BIAS_LAST_STAGE = trim.Trimmer
+BIAS_EXTRA_STAGES = [bias.BiasMasterLevelSubtractor, bias.BiasComparer, nres_bias.BiasMaker]
```

Comment:
On the other hand, if you think the overrides of BNZAI imaging settings are okay, then you could override all of the settings (e.g. change this line to `settings.BIAS_EXTRA_STAGES = [...]`)  and then you wouldn't need to redefine any of the `make_master` methods in `main.py`. You would need to rename `NRES_CRITERIA` though.